### ISSUE TITLE
attendance issue with student names and class list

### ISSUE THREAD

**Issue by swkoehler** [al-caughey/Google-Meet-Attendance#115]
First of all thank you immensely for this app, it has made my teacher routine of taking attendance manageable!  I will be donating as a thank you.
Issue:
I have two different classes with different students each with the last name of Young. In each class, they are no longer recorded as present even though they are in the class.  
Strangely however, a new student appears in each class.  The new student has the same first name, but both have the last name of 'Ng' and are marked present!  There are no students in our school with the last name of 'Ng'.

Perhaps this bug is linked to the fact that both students with the last name of Young are the last students in the alpha sorted list in their respective classes?

**Comment by AlCaughey** [comment]
yeah... there's a bug... but if you go to the facebook page, there are
steps to resolve it at your end
